Source/UI/NewUI/BillingForm/Style/Cell/BillingFormHeaderCellStyle.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+// Header Cell
+public protocol BillingFormHeaderCellStyle {
```

Comment:
Not certain if the comment above the protocol provides any value? The type naming should clearly convey it, and if we want the user of the syntax to flag some helper comment, we should at least use the description marker (`///`)

Reply:
Not going to lie, going ahead with these comments above the class/protocol declaration makes our publicly exposed code look unfinished.

It would be nice if our exposed code would make CKO appear knowledgeable and professional, not showing useless comments. This occurs in a few places, happy to flag them if we decide to do something about it.

Reply:
Fixed